Marlin/example_configurations/Micromake C1/README.md

```diff
@@ -0,0 +1,16 @@
+# Micromake C1
+
+### In the folder "basic"
+Configuration files for Micromake C1 without mods
+  - English LCD 2X16 Characters
+  - Motors 16 STEPS
+  - No heated bed
+  - No probe, etc.
+  - Like a standard C1 as shipped by Micromake.
+
+### In the folder "enhanced"
+Configuration files for Micromake C1 with…
+  - 128 STEPS configured with jumper on the motherboard (all open for 128 Steps).
+  - Capacitive Probe (Adjust offsets at your convenience)
+  - French language with no accents for Japanese LCD.
+  
```

Comment:
The configs should be moved to folders named `Micromake/C1/basic` and `Micromake/C1/basic` instead of `Micromake C1/Naked` and `Micromake C1/metalsearch's C1`.

Please remove all extraneous content from configs and make them as generic as possible. Make the machine names the name of the machine model: "`Micromake C1`. Make the author string much shorter.